dot/rpc/subscription/websocket.go

```diff
@@ -285,6 +287,7 @@ func (c *WSConn) initExtrinsicWatch(reqID float64, params interface{}) (Listener
 		return nil, err
 	}
 
+	// TODO (ed) the importedChan get deleted by unregister imported channel in defer in Listen (listeners.go line 236)
 	// listen for built blocks
 	esl := &ExtrinsicSubmitListener{
```

Comment:
can we move this code to `listeners.go` to a `NewExtrinsicSubmitListener` constructor.  That way we can see the creation and closing of channels all in the same file.

Reply:
Moved code to new NewExtrinsicSubmitListener in listeners.go.